assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -8,7 +8,7 @@
 
 	--color-accent: #{$blue-medium};
 	--color-accent-light: #{$orange-jazzy};
-	// --color-accent-dark: #{};
+	--color-accent-dark: #{darken( $blue-medium, 8% )};
```

Comment:
I'm not sure about this one - that's the previous value taken from the primary button border color. Is that what we want to use here? Alternatively we can override it locally or use a different color altogether /cc @drw158

Reply:
Yes! For the current theme, we're using the exact values that were being used before. This is just so it looks 1:1 with what we're seeing in production. The actual new Classic Blue will be based on Classic Bright once that is done.

Reply:
Yeah, the question for me here is do we want to use `darken( $blue-medium, 8% )` as `--color-accent-dark` which might also be used elsewhere. Are we ok with `darken( $blue-medium, 8% )` as our dark accent color or is it e.g. `darken( $blue-medium, 10% )`?

Reply:
This won't matter so much once Classic Bright ships, right? Our plan is to get rid of the current old color scheme once this ships. Everyone will be on Classic Bright (they can switch to Classic Blue if they want).

Reply:
That's true. I might find an alternative variable name as long as we haven't shipped Classic Bright and the new Classic Blue.